kansha/app/comp.py

```diff
@@ -244,6 +245,7 @@ def set_config(self, config_filename, conf, error):
 
         # search_engine engine configuration
         self.search_engine = SearchEngine(**conf['search'])
+        fts_schema.Card.update_schema(self.card_extensions)
```

Comment:
> Le 12 janv. 2016 à 15:01, Romuald Texier-Marcadé <notifications@github.com> a écrit :
> 
> In kansha/app/comp.py:
> 
> > @@ -244,6 +245,7 @@ def set_config(self, config_filename, conf, error):
> >  
> >          # search_engine engine configuration
> >          self.search_engine = SearchEngine(**conf['search'])
> > +        fts_schema.Card.update_schema(self.card_extensions)
> 
> Most indexers work best with a schema and some just require one.

Of course. The purpose is not to remove the schema but to remove all this `Document`
class/meta-class ORM-like stuff

> 
> —
> Reply to this email directly or view it on GitHub.
>